### PR TITLE
Add testpmd soft link in case new version dpdk-tools change testpmd

### PR DESCRIPTION
**Pull request by wanghekai**
Add testpmd soft link in case new version dpdk-tools change testpmd from testpmd to dpdk-testpmd

### CODE REVIEW

File: ovs_performance.py [hunk 1]
```diff
@@ -4102,6 +4121,7 @@ def main():
     #                     "test_p2v_1000")
     # sys.exit(-1)
 
+
```
Comment (chaudron) [inline ovs_performance.py]: Can you remove this extra line you introduced?

File: ovs_performance.py [hunk 2]
```diff
@@ -4141,6 +4161,10 @@ def main():
     if config.debug_dut_shell:
         dut_shell.logger.setLevel(logging.DEBUG)
 
+    # If new version dpdk-tool changes the command "testpmd" to "dpdk-testpmd"
+    # create soft link.
```
Comment (chaudron) [inline ovs_performance.py]: Can you add the following line before the call?

    lprint("- Create testpmd link on VM for dpdk-testpmd, if needed...")

**Review by chaudron — CHANGES_REQUESTED** [review]
Thanks for the suggested changes, some small style requests are below. Can you please fix them, and I'll merge the request.

File: ovs_performance.py [hunk 3]
```diff
@@ -3462,6 +3462,25 @@ def get_ovs_version():
...
+def create_testpmd_link_if_dpdk_new(vm):
+    cmd = r"sshpass -p {2} ssh -o UserKnownHostsFile=/dev/null " \
+          r"-o StrictHostKeyChecking=no -n {1}@{0} " \
+          r"command -v testpmd". \
+          format(vm, config.dut_vm_user, config.dut_vm_password)
+
+    result = dut_shell.dut_exec('', raw_cmd=['sh', '-c', cmd],
+                                die_on_error=False)
+
+    m = re.search('testpmd',result.output)
```
Comment (chaudron) [inline ovs_performance.py:3474]: Please remove the lprint() from this function:

```
-    if m:
-        lprint("dpdk-tool has command testpmd")
-    else:
+    if not m:
         cmd = r"sshpass -p {2} ssh -o UserKnownHostsFile=/dev/null " \
               r"-o StrictHostKeyChecking=no -n {1}@{0} " \
               r"ln -s /usr/bin/dpdk-testpmd /usr/bin/testpmd". \
```

File: ovs_performance.py (hunk 1 above)
Comment (wanghekai) [reply ovs_performance.py]: NP

File: ovs_performance.py (hunk 2 above)
Comment (wanghekai) [reply ovs_performance.py]: Yes

File: ovs_performance.py (hunk 3 above)
Comment (wanghekai) [reply ovs_performance.py:3474]: Already done

**Comment by chaudron** [comment]
Thanks for the change, added!

**Pull request merged**